pyaedt/edb_core/components.py

```diff
@@ -686,7 +686,8 @@ def deactivate_rlc_component(self, component=None, create_circuit_port=False):
             Reference designator of the RLC component.
 
         create_circuit_port : bool
-            True, Component is converted into circuit port, False is just deactivated
+            Whether to replace the deactivated RLC component to a circuit port. The default
+            is ``False``.
 
```

Comment:
The first sentence either needs to be one of the following:
"Whether to convert the deactivated RLC component to a circuit port."
"Whether to replace the deactivated RLC component with a circuit port."

Reply:
Thanks @PipKat :)